### DISCUSSION TITLE
Jellyfin - Passthrough of HDDs

### DISCUSSION THREAD

**Discussion by TLHD** [tteck/Proxmox#2042] [Help Needed]
I hope nobody shoots me for this one (first time user, linux noob): 
Proxmox installed fine, so did Jellyfin with the script. 
I thought it would be easy to passthrough my media hdds with 

> "qm set 101 -sata1 /dev/disk/by-id/ata-TOSHIBA_HDWG31E_Y1G0A06DFUWH"
 for example. 
Seems like i was wrong: 

> root@pve:~# qm set 101 -sata1 /dev/disk/by-id/ata-TOSHIBA_HDWG31E_Y1G0A06DFUWH
> Configuration file 'nodes/pve/qemu-server/101.conf' does not exist

any way to fix that?

**Comment by tteck** [comment]
`qm` is the tool to manage virtual machines, not containers.

**Reply by TLHD** [reply]
Thank you for the fast reply. 
Ok, got that. 
Could you also help me/give me a hint how/if that´s possible for containers then?

**Reply by tteck** [reply]
https://pve.proxmox.com/wiki/Linux_Container#_bind_mount_points
